docs/quickstart/custom_dataset.md

```diff
@@ -107,3 +107,27 @@ cd vcpkg
 
 :::::
 ::::::
+
+## Record3D Capture
+
+Nerfstudio can also be trained directly from >=iPhone 12 Pro captures from the [Record3D app](https://record3d.app/). This uses the iPhone's LiDAR sensors to calculate camera poses, so COLMAP is not needed. 
+
+Record a video and export with the EXR + JPG sequence format. 
+
+```{image} imgs/record_3d_video_selection.png
```

Comment:
huh on the build page i dont actually see this image show up?

Reply:
yeah I can't seem to get them to show either when I build. It shows in vscode though :/